content/20.pharmaceuticals.md

```diff
@@ -436,19 +436,26 @@ Additionally, while TCZ is not yet as established, this therapy suggests that th
+Because the drug development process is slow and costly and the development of compounds specifically targeted to an emerging viral threat are not a practical short-term solution, efforts to screen existing drug compounds for alternative indications have become popular [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1111/bph.13895; @doi:10.1039/D0CS01065K; @doi:10.1016/j.coi.2013.08.004].
 High-throughput screening has been a goal of pharmaceutical development since at least the mid-1980s [@doi:10.1038/sj.bjp.0707373].
-Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use high-throughput computational approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
-Computational drug repurposing screens are able to take advantage of big data in biology [@doi:10.1038/nrd.2018.168] and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
-Advancements in robotics also facilitate the experimental component of high-throughput screening [@doi:10.1039/D0CS01065K].
+Traditionally, phenotypic screens were used to test which compounds would induce a desired change in _in vitro_ or _in vivo_ models, focusing on empirical, function-oriented exploration naïve to molecular mechanism [@doi:10.1126/scitranslmed.aab1201; @doi:10.1038/clpt.2012.236; @doi:10.1038/nrd4366].
+Around the turn of the millennium, advances in molecular biology allowed for HTS to shift towards emphasizing specific molecular targets that were hypothesized to have a desired effect and to screen compounds for those interacting with the target.
+In the age of biological big data, _in silico_ studies have emerged as a new frontier [@doi:10.1016/j.coi.2013.08.004].
+
+Today, some efforts to screen compounds for potential repurposing opportunities are experimental, but others use computational HTS approaches [@doi:10.1016/j.drudis.2019.06.014; @doi:10.1016/j.tim.2018.04.004].
+Computational drug repurposing screens can take advantage of big data in biology [@doi:10.1038/nrd.2018.168] and as a result are much more feasible today than during the height of the SARS and MERS outbreaks in the early 2000s and early 2010s, respectively.
+Advancements in robotics also facilitate the experimental component of HTS [@doi:10.1039/D0CS01065K].
 For viral diseases, the goal of drug repurposing is typically to identify existing drugs that have an antiviral effect likely to impede the virus of interest.
 While both small molecules and biologics can be candidates for repurposing, the significantly lower price of many small molecule drugs means that they are typically more appealing candidates [@doi:10.1089/adt.2019.950].
 Depending on the design, screens can be either hypothesis driven or hypothesis free.
 
-#### Hypothesis-Driven Computational Analyses
+#### Hypothesis-Driven Screening
```

Comment:
Above we introduced the distinction between target- versus phenotypic-based screens.  Is target-based equivalent to hypothesis-driven, or is hypothesis-driven versus hypothesis-free a separate classification?

Reply:
I think that phenotypic screens could be hypothesis-guided or hypothesis-free depending on the library of compounds used. I agree with @rando2's interpretation above though - mostly as broad as possible.

Reply:
I think with my latest commit (0a02366a), I've (hopefully) moved things around enough to keep the focus on computational screens, with the context from the target-based vs phenotypic screen debate brought in just to highlight the importance of unifying hypothesis-based and discovery-driven methods.